client/components/single-song.js

```diff
@@ -12,9 +18,13 @@ class Single_Song extends React.Component {
     }
   }
 
+  handleClick(event) {
+    console.log('EVENT', event)
```

Comment:
it's better to avoid pushing the code into the main branch that has any console logs in it